castai/delta_types.go

```diff
@@ -31,6 +31,8 @@ type DeltaItem struct {
 	// ObjectContainers and ObjectStatus are set only for objects which could contain containers.
 	ObjectContainers []Container `json:"object_containers,omitempty"`
 	ObjectStatus     interface{} `json:"object_status,omitempty"`
+	// ObjectImagesChanged is set to true when previous container status contained different image set.
+	ObjectImagesChanged bool `json:"object_images_changed,omitempty"`
```

Comment:
What happens if kvisor is not running for some time and image changed? If this logic is important in backend I think it's better to detect image change from actual db state.